tools/cldr/cldr-to-icu/src/test/java/org/unicode/icu/tool/cldrtoicu/PseudoLocalesTest.java

```diff
@@ -75,12 +79,12 @@ public void testBidi() {
             .addLocaleData("en",
                 value(included, "{Hello} {0} {World} 100x"),
                 value(pattern, "'plus' HH:mm; 'minus' HH:mm"))
-            .addInheritedData("en",
-                value(inherited, "UTC"));
+            .addLocaleData("en_001", value(inherited, "UTC"))
```

Comment:
the "inherited" value appears as part of the pseudo locale data, but the "root" data doesn't (this is even true for normal values that could be pseudo-localized). One day we might revisit this and allow some root values to be pseudo-localized, but this is pretty good now (and the generated data doesn't include aliases any more).